src/apps/Hosting/HostingListItem.js

```diff
@@ -114,8 +114,8 @@ class HostingListItem extends Component {
             />
             <MenuItem
               onTouchTap={showPublishDialog}
-              primaryText="Publish"
-              data-e2e="dropdown-hosting-item-publish"
+              primaryText="Set as default"
+              data-e2e="dropdown-hosting-item-set-default"
```

Comment:
@wpasinski not used in tests (right now) so only updated text, but it will use it in future